src/hooks/progress/use-collection-progress.ts

```diff
@@ -23,15 +28,22 @@ interface UseCollectionProgressResult
 function useCollectionProgress({
 	collectionId,
 }: UseCollectionProgressOptions): UseCollectionProgressResult {
-	// Get the current user's access token
+	/**
+	 * Get the current user's access token, and the batch query status.
+	 * We enable the query only if:
+	 * - an accessToken is present
+	 * - a batch query, if one exists, has been attempted
+	 */
 	const { session } = useAuthentication()
 	const accessToken = session?.accessToken
+	const hasWaitedForBatchQuery = useHasWaitedForQuery([PROGRESS_BATCH_QUERY_ID])
```

Comment:
Have updated `use-collection-progress` to wait on any progress batch query before being enabled (if a batch query exists - if not, we don't wait).